lavalab-gen.py

```diff
@@ -155,6 +156,12 @@ def main():
         groupdir = "%s/groups" % workerdir
         os.mkdir(groupdir)
         worker = master
+        if "version" in worker:
+            dockerfile = open("%s/Dockerfile" % workerdir, "r+")
+            dockerfilec = re.sub('(^FROM.*:).*', '\g<1>%s' % worker["version"], dockerfile.read())
+            dockerfile.seek(0)
```

Comment:
This smells funny.  What if the new version string has more characters than the previous version string.   Aren't you going to potentially overwrite stuff in the updated Dockerfile?

Reply:
I think the changelog needs more description on what exactly is being replaced in the Dockerfile

Reply:
I have added an example.
I have tested, and there no risk of overwrite. (the regex seems to stop at newline)